.github/workflows/nightly.yml

```diff
@@ -0,0 +1,55 @@
+name: Nightly release
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 23 * * *"
+
+jobs:
+  publish_release:
+    name: Publish release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set version
+        run: |
+          echo "RUSTC_PERF_VERSION=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
+
+      - name: Checkout the source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: 1.66.0
```

Comment:
Is pinning this the right decision? Should we use stable here?

Reply:
Indeed `stable` will be better, to regularly check that we can still build on it.